### PR TITLE
Priyanka_Assignment_22

### PR DESCRIPTION
**Pull request by PriyankaKOct2020**
Updated Logic for Exam3:
image

Assignment 22

image

### CODE REVIEW

File: src/priyanka/Assignment22/PrimeNumbers.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100.
+*/
+public class PrimeNumbers {
+
+	void primeNo(int startNum, int endNum) {
+
+		for (int index = startNum; index <= endNum; index++) {
+			int count = 0;
+			for (int innerIndex = 1; innerIndex <= endNum; innerIndex++) {
```
Comment (harshvegada) [inline src/priyanka/Assignment22/PrimeNumbers.java:14]: innerIndex should start from 2 right,
we cannot identify 1 as prime or non-prime
this is correction is require

File: src/priyanka/Assignment22/PrimeNumbers.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+				if (index % innerIndex == 0) {
+					count++;
```
Comment (harshvegada) [inline src/priyanka/Assignment22/PrimeNumbers.java:16]: once you are in if condition you should break right, bcoz now you knwo this number is not prime number

pls refer this code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/688/files

**Review by harshvegada — CHANGES_REQUESTED** [review]
Pls accommodate require change,

File: src/priyanka/Assignment22/PrimeNumbers.java (hunk 1 above)
Comment (PriyankaKOct2020) [reply src/priyanka/Assignment22/PrimeNumbers.java]: Hi Harsh, 
I have updated the code, please review.

**Review by harshvegada — APPROVED** [review]
Code Looks Good to me, Good to merge!

**Pull request merged**